### PR TITLE
Add rules_android v0.5.0

### PR DESCRIPTION
**Pull request by ted-xie**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (rules_android) have been updated in this PR. Please review the changes.

**Comment by ted-xie** [comment]
In the  (passed) BazelCI run, I skipped the "Wait on BCR Maintainer Review" step to get a CI result faster, hope that's OK.

**Review by Wyverald — APPROVED** [review]

**Pull request merged**
